README-sycl.md

```diff
@@ -17,14 +17,14 @@ resources for GPUs.
 
 Currently, only one example works on NVIDIA SM 80.
 
-## Building with SYCL support
-
-To build CUTLASS SYCL support you need the latest version of DPC++ compiler, 
-you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
+## Requirements
+ 
+To build CUTLASS SYCL support you need the latest version of DPC++ compiler, you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
 or build the compiler from source.
 In either case, make sure to enable the NVIDIA plugin so you can build applications
```

Comment:
I do not suppose the Nvidia / AMD plugins can be installed unless the base toolkit is installed.
For this we need to build dpcpp from source with the Cuda backend enabled

Reply:
No you dont need to you can the plugins from codpelay nightly build.

Reply:
I see, in that case let's not call it plugins, which confuses with the Plugins available on the codeplay's website to people who are completely new to SYCL,

we can phrase it as - 
```suggestion
To build CUTLASS with SYCL support, install the latest DPC++ compiler with the CUDA backend enabled, either by building from source as described [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda) ,  or by downloading the [nightly releases](https://github.com/intel/llvm/releases)
```